### PR TITLE
Updated WW3's Configuration: Move Variables from ww3_shel.nml to wav_in

### PR DESCRIPTION
**Pull request by ezhilsabareesh8**
This PR addresses issue [#143](https://github.com/COSIMA/access-om3/issues/143) which is about the recent update in WW3's `wav_shel_inp.F90` codebase. The `ww3_shel.nml` file is no longer utilised and in this PR all variables stored in `ww3_shel.nml` are moved to the `wav_in` file for proper functionality of the configuration.

### CODE REVIEW

**Comment by anton-seaice** [comment]
Are all the variables used?

Like there was this cryptic note: 

`
! Note that CESM restarts are not controlled by namelist input
! - but rather by driver config variables
`
which I guess applies to the 'domain' lines and implies they are not used?

**Comment by ezhilsabareesh8** [comment]
> which I guess applies to the 'domain' lines and implies they are not used?

Thanks Anton, Yes, only the `domain_nml` is not used, I have removed those in this commit [ffa0f0d](https://github.com/COSIMA/MOM6-CICE6-WW3/pull/23/commits/ffa0f0de1392bbe5f1cf14a013a27fdf2382340f)

File: wav_in [hunk 1]
```diff
@@ -5,3 +5,31 @@
...
+  input%forcing%ice_param1   = 'C'
+  input%forcing%ice_param5   = 'C'
+/
+&output_type_nml
+  type%field%list = 'WND ICE HS T02 T0M1 T01 FP DIR EF USS TOC IC1 IC5'
+  type%point%file = 'ww3_points.list'
+/
+&output_date_nml
+  date%field%outffile  = '1'
+  date%field%start     = '19990101 0'
```
Comment (anton-seaice) [inline wav_in:22]: Sorry - i missed this the first time. Should this be 1958 ?

File: wav_in [hunk 2]
```diff
@@ -5,3 +5,31 @@
...
+  date%field%stride    = '86400'
+  date%field%stop      = '99990101 0'
```
Comment (anton-seaice) [inline wav_in:24]: And this be 2019? (Although maybe it will cycle around, so leaving the default is better?)

**Review by anton-seaice — COMMENTED** [review]
It looks like we forgot to do #15 for this repo?

File: wav_in (hunk 1 above)
Comment (ezhilsabareesh8) [reply wav_in]: Fixed in this commit 33fa3c6

File: wav_in (hunk 2 above)
Comment (ezhilsabareesh8) [reply wav_in]: Changed in this commit e8fd445

**Comment by ezhilsabareesh8** [comment]
> It looks like we forgot to do #15 for the iaf branch?

I have changed the WW3 inputs, only `ice_in` needs to be updated. We merge this and handle in separate PR or would you like do it  in this PR?

**Comment by anton-seaice** [comment]
In a seperate PR :)

**Review by anton-seaice — APPROVED** [review]
I ran this and it appears to work.

I confirmed that the time stamp in the gridded output looks correct:

```
        double time(time) ;
                time:units = "seconds since 1958-01-01 00:00:00" ;
                time:calendar = "standard" ;

 time = 86400 ;
```

We should soon do a review across the two WW3 and the non-WW3 configs for consistency between configs.

**Review by micaeljtoliveira — APPROVED** [review]
LGTM!

**Pull request merged**